lib/util/git/url.spec.ts

```diff
@@ -77,6 +77,14 @@ describe('util/git/url', () => {
         'https://x-access-token:token@github.com/some/repo'
       );
     });
+
+    it('removes username/password from URL', () => {
+      expect(getHttpUrl('https://user:password@foo.bar/')).toBe('https://foo.bar/');
```

Comment:
The url parse seems to take the `user` into the parsed url if no repo is specified
```suggestion
      expect(getHttpUrl('https://user:password@foo.bar/someOrg/someRepo')).toBe('https://foo.bar/someOrg/someRepo');
```